### PR TITLE
Abstract colour out of hosted page model

### PR DESCRIPTION
**Pull request by kelvin-chappell**
Because there's nothing font-specific about it.

### CODE REVIEW

File: common/test/common/commercial/hosted/ColourTest.scala [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+/*
+ * For colours, see http://www.colorhexa.com/<hexcode>
+ * eg. http://www.colorhexa.com/2ec869
+ */
+class ColourTest extends FlatSpec with Matchers {
+
+  "isDark" should "be true for Zootropolis green" in {
+    val zootropolisColour = Colour("#2ec869")
+    zootropolisColour shouldBe 'dark
```
Comment (lps88) [inline common/test/common/commercial/hosted/ColourTest.scala]: this syntax is weird! Does it work using the `isDark` val?

Comment (kelvin-chappell) [inline common/test/common/commercial/hosted/ColourTest.scala]: Yes. http://www.scalatest.org/user_guide/using_matchers#checkingBooleanPropertiesWithBe

(But if it makes the test worse, I'll get rid of it.)

Comment (lps88) [inline common/test/common/commercial/hosted/ColourTest.scala]: Nah I think it's fine, just haven't seen it before :)

**Comment by lps88** [comment]
👍

**Pull request merged**